shared/productTypes.ts

```diff
@@ -77,8 +77,7 @@ export type AllProductsProductTypeFilterString =
 	| 'Digipack'
 	| 'SupporterPlus'
 	| 'ContentSubscription'
-	| 'GuardianPatron'
-	| 'RecurringSupport';
```

Comment:
quick question why are we removing RecurringSupport?

Reply:
Good question. There's a filter function in MDAPI [here](https://github.com/guardian/members-data-api/blob/9efe6e81309ccf90639754d4affd8c8af2d6f5fe/membership-attribute-service/app/services/AccountDetailsFromZuora.scala#L194) which matches on the different strings and `RecurringSupport` is not one of them